### ISSUE TITLE
Is depthwise conv / group conv supported on inferentia?

### ISSUE THREAD

**Issue by zqliu19** [aws/aws-neuron-sdk#214]
Is depthwise conv / group conv supported on inferentia for networks like MobileNet and ResNeXt?

**Comment by aws-taylor** [comment]
Hello zqliu19,
  
By their nature, neural networks optimized for embedded applications like MobileNet will see limited benefit by using Inferentia. That being said, we are actively working on adding support for these operators to improve our customer experience. I encourage you to watch our [release notes](https://github.com/aws/aws-neuron-sdk/tree/master/release-notes) for announcements in the coming months for updates regarding operator support.

Please let me know if there is anything else with which I can help.

-Taylor

**Comment by zqliu19** [comment]
> Hello zqliu19,
> 
> By their nature, neural networks optimized for embedded applications like MobileNet will see limited benefit by using Inferentia since they optimize for lower computational execution. For example, we hear from customers that running ResNet on Inferentia provides same throughput as MobileNet, but with better accuracy. That being said, we are actively working on adding support for these operators to improve our customer experience. I encourage you to watch our [release notes](https://github.com/aws/aws-neuron-sdk/tree/master/release-notes) for announcements in the coming months for updates regarding operator support.
> 
> Please let me know if there is anything else with which I can help.
> 
> -Taylor

Thanks a lot Taylor, 
What about group conv in ResNeXt? Is group conv supported now and do we have plans to support group conv out of the box in the future?

**Comment by aws-taylor** [comment]
Hello zqliu19,

   Neither depth-wise nor group convolution are supported at this time, but we are tracking them since we know they're important for networks like ResNeXt and some EfficientNet variants. 

Regards,
Taylor

**Comment by aws-joshim** [comment]
Hello zqliu19, we are planning to add support for group convolution with the Pytorch based ResNext model. We encourage you to “Watch” the associated roadmap item for it @ https://github.com/aws/aws-neuron-sdk/issues/226 for progress updates. We also encourage +1 to these roadmap items to let us know what would be helpful

**Comment by awsrjh** [comment]
Hello zqliu19,

we have released support for ResNext for Pytorch  in our most recent release. Please see this link for more information:

https://awsdocs-neuron.readthedocs-hosted.com/en/latest/release-notes/index.html

Please let us know if this is helpful or if you have any additional needs by reopening this issue or starting a new one.

best regards.
